tcms/testplans/static/testplans/js/mutable.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready((function() {
+    update_version_select_from_product();
+
+    $('#id_product').change(update_version_select_from_product);
```

Comment:
Change to:

```
    $('#id_product').change(function() {
        $('#id_version').find('option').remove();
        update_version_select_from_product();
    });
```

to resolve the problem with the first element not being removed.